apps/src/applab/applab.js

```diff
@@ -1351,7 +1374,7 @@ Applab.serializeToLevelHtml = function () {
   var divApplab = document.getElementById('divApplab');
   var clone = divApplab.cloneNode(true);
   // Remove unwanted classes added by jQuery.draggable.
-  $(clone).find('*').removeAttr('class');
+  $(clone).find('*').removeClass('ui-draggable ui-draggable-handle');
```

Comment:
This doesn't quite work, for a somewhat interesting reason. When we create our clone, the data associated with elements does not come over with it. I believe in previous version of jQuery, data was stored on the elements themselves in the form of an attribute data-<type>="my data here". However, in the latest versions this is stored in $.cache, which apparently does not copy itself when cloned.

I think the solution is to just replace line 2 with a check for hasClass('ui-draggable'), and then still use destroy to get rid of all relevant classes.